### DISCUSSION TITLE
How to keymap "X" to "unselect_line_above"?

### DISCUSSION THREAD

**Discussion by HairlessVillager** [helix-editor/helix#11389] [Q&A] [answered]
My expected behaivor is like "X" will not select the current line and move the cursor up, just like undo a previous "x" operation.

I've tried a keymap: `""X" = ["select_mode", "move_visual_line_up", "goto_line_end", "exit_select_mode"]
`. But it not works as expected.

**Comment by bogdanataffirm** [comment, marked as answer]
I have these 2 settings:
```
X = "select_line_above"
x = "select_line_below"
```
They work exactly as you want, with the added bonus that you can do that in reverse: you can start to select lines upwards (with `X`), then deselect them downwards (with `x`).

**Reply by HairlessVillager** [reply]
Yes! Thank you🥰🥰🥰
